front/src/filters/filters.js

```diff
@@ -0,0 +1,8 @@
+import Vue from 'vue';
+
+Vue.filter('getImgSrc', function (imgPath, baseUrl) {
```

Comment:
serverImagePath/serverImageSrc
fullImagePath
not get - it is filter - it does't require get in name and it is not correct